docs/resources/rds_read_replica_instance.md

```diff
@@ -81,6 +85,20 @@ The following arguments are supported:
 * `enterprise_project_id` - (Optional, String, ForceNew) The enterprise project id of the read replica instance.
   Changing this parameter will create a new resource.
 
+* `charging_mode` - (Optional, String, ForceNew) Specifies the charging mode of the RDS DB instance. Valid values are
+  *prePaid* and *postPaid*, defaults to *postPaid*. Changing this creates a new resource.
+
+* `period_unit` - (Optional, String, ForceNew) Specifies the charging period unit of the RDS DB instance. Valid values
+  are *month* and *year*. This parameter is mandatory if `charging_mode` is set to *prePaid*. Changing this creates a
+  new resource.
+
+* `period` - (Optional, Int, ForceNew) Specifies the charging period of the RDS DB instance. If `period_unit` is set
+  to *month*, the value ranges from 1 to 9. If `period_unit` is set to *year*, the value ranges from 1 to 3. This
+  parameter is mandatory if `charging_mode` is set to *prePaid*. Changing this creates a new resource.
+
+* `auto_renew` - (Optional, String) Specifies whether auto renew is enabled. Valid values are "true" and "
+  false". Changing this creates a new resource.
```

Comment:
please remove this: `Changing this creates a new resource.`

Reply:
done